### PR TITLE
radio: Check coverage tests

### LINKED ISSUE #589: radio: Check coverage tests

### PR DESCRIPTION
**Pull request by sipayRT**
Close #589

### CODE REVIEW

**Comment by sipayRT** [comment]
@dfilatov посмотри, плз

File: common.blocks/radio/_type/radio_type_button.spec.js [hunk 1]
```diff
@@ -45,6 +45,19 @@ describe('radio_type_button', function() {
             radioOption.findBlockOn('button').hasMod('disabled').should.be.false;
         });
     });
+
+    describe('events', function() {
```
Comment (dfilatov) [inline common.blocks/radio/_type/radio_type_button.spec.js]: I would call this section "focused" by analogy with "checked" and "disabled" sections above.

File: common.blocks/radio/_type/radio_type_button.spec.js [hunk 2]
```diff
@@ -45,6 +45,19 @@ describe('radio_type_button', function() {
...
+        it('should set mod focused on pointerclick', function() {
```
Comment (narqo) [inline common.blocks/radio/_type/radio_type_button.spec.js]: Maybe 

> should set "focused" mod on click

**Comment by sipayRT** [comment]
@dfilatov @narqo rebased and fixed

**Comment by dfilatov** [comment]
lgtm

**Pull request merged**